mods/ra/maps/soviet-09/rules.yaml

```diff
@@ -0,0 +1,121 @@
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, soviet09.lua, soviet09-AI.lua
+	MissionData:
+		BriefingVideo: soviet9.vqa
+		WinVideo: v2rocket.vqa
+		LossVideo: sfrozen.vqa
+		StartVideo: movingin.vqa
+		Briefing: The Allied forces have intercepted and destroyed a convoy that carried parts for our secret weapon. One truck remains, but they have captured that last truck and its cargo.\n\nThis is not acceptable! You are to destroy that truck before the Allies leave the area with it.
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+	SmudgeLayer@SCORCH:
+		InitialSmudges:
+			98,87: sc3,0
+			100,87: sc2,0
+			108,87: sc1,0
+			88,93: sc6,0
+			104,101: sc3,0
+	SmudgeLayer@CRATER:
+		InitialSmudges:
+			99,87: cr1,2
+			86,93: cr1,0
+			87,93: cr1,1
+
+Player:
+	PlayerResources:
+		DefaultCash: 8000
+
```

Comment:
[On IRC, it was mentioned that the ending FMV cuts in too soon](https://logs.openra.net/?year=2020&month=07&day=05#23:41:30).  If you want you can increase the delay with the GameOverDelay setting on MissionObjectives like so:

```yaml:
Player:
	MissionObjectives:
		GameOverDelay: 3500
```

Reply:
I expect that this will impact other missions too?

IMO would be best to address this in a separate PR, maybe by adding this to the campaign-rules or even changing the default on the trait?

Reply:
> IMO would be best to address this in a separate PR, maybe by adding this to the campaign-rules or even changing the default on the trait?

Agreed, I've noticed this being abrupt in most if not all missions.